tasks/coverage/snapshots/minifier_test262.snap

```diff
@@ -2,4 +2,9 @@ commit: d62fa93c
 
 minifier_test262 Summary:
 AST Parsed     : 43914/43914 (100.00%)
-Positive Passed: 43914/43914 (100.00%)
+Positive Passed: 43909/43914 (99.99%)
+Compress: tasks/coverage/test262/test/language/directive-prologue/10.1.1-6-s.js
```

Comment:
👀

Reply:
This is actually unrelated to `CollapseVariableDeclarations` because I checked the relevant cases, and it didn't make any modifications to them. I need to investigate the cause further.

Reply:
```rust
Mismatch("Compress", "var x = 0;\nvar x = {};\n", "var x = 0, x = {};\n")
```

I believe this is due to the fact that during the first run, we executed the removal of unnecessary code first, without executing `CollapseVariableDeclarations`. @7086cmd cc

https://github.com/oxc-project/oxc/blob/702c049ebc3bb873d4c2be8eeb247f34bb551962/crates/oxc/src/compiler.rs#L186-L187

https://github.com/oxc-project/oxc/blob/702c049ebc3bb873d4c2be8eeb247f34bb551962/crates/oxc_minifier/src/compressor.rs#L40-L46

https://github.com/oxc-project/oxc/blob/702c049ebc3bb873d4c2be8eeb247f34bb551962/tasks/coverage/src/driver.rs#L126-L141

Reply:
Ok, I'll come back to this PR once I'm done with the other ones.

Reply:
Yes, it seems. In this scenario, actually the test runner joins the var (actually we don't want them to join!).

Reply:
But I havn't eliminate this dead code right now! 🤯